network/p2p/middleware.go

```diff
@@ -499,7 +499,10 @@ func (m *Middleware) Subscribe(channel network.Channel) error {
 	var validators []psValidator.MessageValidator
 	if !engine.PublicChannels().Contains(channel) {
 		// for channels used by the staked nodes, add the topic validator to filter out messages from non-staked nodes
-		validators = append(validators, psValidator.StakedValidator(m.ov.Identity))
+		validators = append(validators,
+			psValidator.StakedValidator(m.ov.Identity),
```

Comment:
I think we can remove this validator. All it does is check that the identity exists, which `AuthorizedSenderValidator` will also do.

Reply:
I like the idea of keeping it for the sake of readability. I can look at this middleware.go file and automatically deduce that being a staked node is a requirement for non-public channels. What do you think ?

Reply:
For readability, I'm more inclined to include that in the documentation for `AuthorizedSenderValidator`, or use a comment here. Using both `AuthorizedSenderValidator` and `StakedValidator` separately could also lead the reader to incorrectly infer that an "authorized sender" may _not_ be staked.

Reply:
Okay that makes sense i'll add a comment

Reply:
https://github.com/onflow/flow-go/pull/2317/commits/9920d68c30fdb7d362ee86723c1ffc4babeb62e0